src/readFields.js

```diff
@@ -7,6 +7,7 @@ import isPristine from './isPristine';
 import isValid from './isValid';
 import getValues from './getValues';
 import silencePromise from './silencePromise';
+import {get} from 'lodash';
```

Comment:
If you want to reduce the size of the generated bundle, use:
import get from 'lodash/object/get';